### DISCUSSION TITLE
Request for Support with Renatio Form Builder Integration in OFFLINE Boxes (Error 406)

### DISCUSSION THREAD

**Discussion by Andrea-Tecnotrade** [OFFLINE-GmbH/oc-boxes-support#153] [Help]
Dear OFFLINE Boxes Team,

My name is Andrea from Tecnotrade, and I’m using the OFFLINE Boxes plugin in a project based on OctoberCMS. Currently, I have successfully integrated several Renatio Form Builder forms dynamically within the Boxes blocks to handle forms in the pages.

However, I’m encountering an issue when trying to submit a form within a Boxes page: the submission fails, and I receive a 406 error.

I’m reaching out to ask if you have any suggestions or guidance on how to resolve this issue. Is there a specific configuration required to ensure that form submissions work correctly in this context, or are there additional steps needed to avoid the error?

Thank you for your support and for the great work you do. I look forward to your response.

Best regards,
Andrea

**Comment by tobias-kuendig** [comment]
@Andrea-Tecnotrade the Boxes plugin renders a simple CMS page and uses normal Partials to do this. This means there is no special config required.

Depending on the plugin, it might be required, that you add components to the layout. Otherwise the event handlers are not found:
https://docs.boxes.offline.ch/use-cases/components.html#adding-components

Also, it would be useful to know how you integrated the form builder plugin's component and what the actual response body for your 406 status response is.
